lib/inbox.rb

```diff
@@ -84,7 +83,7 @@ def self.interpret_response(result, result_content, options = {})
 
 
   class API
-    attr_accessor :api_server
+    attr_accessor :api_server, :stream_handler
```

Comment:
For consistency reasons, can you move the `:stream_handler` declaration to the next line?

e.g: 
```ruby
attr_accessor :api_server
attr_accessor :stream_handler
```